form.py

```diff
@@ -181,12 +180,12 @@ def confirm_user(self):
         else:
             return False
 
-        if done == 0:
```

Comment:
I think it's safer to leave it as checking for 0, as only in the case of done being 0 should this pass. Adjusting it to if done ==1 may allow bad data to pass the test, unlikely as that may be.